test/evergreen.yml

```diff
@@ -172,7 +172,8 @@ functions:
         set -o errexit
         set -o verbose
         for i in $(seq ${times|1}); do
-          ${test_env_vars|} ${format_test_setting|} ./format.sh ${smp_command|} ${format_test_script_args|} 2>&1
+          ${format_test_setting|}
```

Comment:
The value of $format_test_setting variable is a command i.e, "ulimit -c unlimited" and should be executed as a single line command.